### ISSUE TITLE
Roland s760 gotek c'ant communicate message!

### ISSUE THREAD

**Issue by shindoja** [keirf/flashfloppy#760]
Hi ! A have a s760 with a Gotek booting fine with 000.IMG :)
But i cannot load samples .IMG file its said "cant communicate" on the s760 and error 31 on the Gotek.
What im i missing ? Thanks a lot

**Comment by tenuke** [comment]
https://github.com/keirf/FlashFloppy/wiki/Error-Messages

error 31 = Bad or Unsupported image file

ke 15. helmik. 2023 klo 19.44 shindoja ***@***.***) kirjoitti:

**Comment by shindoja** [comment]
Yep a saw that ! But still ? Its the .IMG a found on that link : https://onedrive.live.com/?authkey=%21AFBw7J%2D2SF9Fr30&id=D541D8022D453E0C%211749&cid=D541D8022D453E0C&parId=root&parQt=sharedby&parCid=516454F7667BB874&o=OneUp

And : http://llamamusic.com/s50s550/unusual_samples.html

**Comment by tenuke** [comment]
oh the .OUT is listed as an supported format but anyways, because the FF
gives you this error, I would still go forward by trying
to convert the images :o

ke 15. helmik. 2023 klo 20.36 Mikko Nurmesniemi ***@***.***)
kirjoitti:

**Comment by shindoja** [comment]
Ok ! Still doesnt work hehe
I changed the extension to IMG ang to OUT. 
I update to 4.7 but blanc screen after the countdown
I came back to 3.38 and it booting again
But when i try to load its doesnt see and say : Please insert disk 

The thing is im a musician haha I dont know much about computer or even de 760 as i just got it

So really maybe there is something really stupid a dont get hehe

Do i have to make a special partition on my usb key for s760 to read the sounds ?

And also , on the Gotek, There is only 4 " parts " a can click on : 000, 001, 002, 003 

Thanks you so much !

**Comment by keirf** [comment]
Can you share a non-working IMG file on this issue ticket?

**Comment by shindoja** [comment]
![Screen Shot 2023-02-20 at 12 56 29 PM](https://user-images.githubusercontent.com/118574645/220174096-da7eabaf-3335-4b0d-bda2-5fbbd899c2dc.png)

**Comment by shindoja** [comment]
Oups it says that

**Comment by shindoja** [comment]
![002](https://user-images.githubusercontent.com/118574645/220174359-03938e4c-0ea0-493e-af68-eafe75fb8e49.jpeg)

I just rename it Jpeg to paste it here, but its the IMG file that i tried to play in the S760 :) Thank you

**Comment by tenuke** [comment]
Try to make an zip-package of it first, then drag&drop that .zip here :)
WinZip for example works for zipping

**Comment by shindoja** [comment]
[002.IMG.zip](https://github.com/keirf/flashfloppy/files/10787349/002.IMG.zip)

**Comment by shindoja** [comment]
Thanks for the tip !

**Comment by shindoja** [comment]
Anybody has a clue ? When i go to dick it say " please insert disk " , Its like i would have to make a partition or something , my usb key is 16g
Thank you !

**Comment by keirf** [comment]
IMG file looks fine.

Maybe you need to create a FF.CFG text file in your USB stick with a different interface setting in it. It's just a plain text file. Eg you could try putting this in it:
```
interface = jppc-hdout
```

**Comment by shindoja** [comment]
Ok i'll try that ! Thanks :)

**Comment by shindoja** [comment]
Oups does not boot anymore. And i tried to revert to shugart and does not boot either. There is definitely something i dont get in the general structure of the Gotek protocoles hehe. I did eventually push the 2 buttons to try reseting..may be i took it back to stock parameters ? 

So from the start what i understand: 

1- Unbox the gotek
2- Put the J3 jumpers to install firmware
3- Use artery to install firmware into Gotek via usb to usb cable
4- Put Gotek physically into the s760
5- Format usb stick in fat 32 ( 16gig)
6- Put s760224 file from roland on the root folder 

That's how i got to boot the s760 until now but never was able to save or read a sample.

Thank you ! Cant wait to play music haha

**Comment by shindoja** [comment]
Could it be that im installing the wrong hex file in the Gotek ? The file installed is flashfloppy t415-st105-3.38.hex 

Im trying to update to a HxCFEUSB_5_1_1_1aRoland_QuickDisk.UPD but the Gotek wont update nothing it seems.

**Comment by shindoja** [comment]
Ok so looks like I have to put a resistor on pin 2 and 34 hehe come back with details !

**Comment by shindoja** [comment]
Is it normal also it does not keep the .CFG file ?

**Comment by keirf** [comment]
Which CFG file?

**Comment by shindoja** [comment]
Also I did the resistor thing on pins 2 and 34 and it did not change nothing hehe

**Comment by shindoja** [comment]
Also its like the gotek does not want to update files when I put them on the USB stick and hold the 2 buttons at start. Its says upd but different error code after that.

**Comment by keirf** [comment]
The config file you write to should be called FF.CFG. Are you writing to IMAGE_A.CFG?

**Comment by shindoja** [comment]
A yes as it was the .CFG file that the s760 created i though its was the right one !
Ok so i'll try whit the FF one. Also : 

1- if you could tell me if I should let the resistors or remove them that would help remove some shaky parameters from the overall situation :)

2- What firmware should i put to be sure now i flashed the : at415-st105-3.39.hex

3- What could be the issue with the update protocol ?

Thanks infinitely

**Comment by keirf** [comment]
I wouldn't mess with the onboard resistors.

What model is your Gotek (written on the PCB)?

**Comment by shindoja** [comment]
ok ! 

Model : SFRKC30.AT2

Also it only boot with : interface = jppc-hdout or IBM PC

**Comment by shindoja** [comment]
Also we are talking on facebook haha

**Comment by keirf** [comment]
Do you have OLED display? What version is shown when you enter update mode?

**Comment by shindoja** [comment]
No oled just stock version ( plan to do that later )

**Comment by keirf** [comment]
You have flashed the correct firmware. Update should work. What's the error.

**Comment by shindoja** [comment]
Most of the time 34 sometimes 01

**Comment by keirf** [comment]
E01 is no update file found. E34 I think shouldn't be possible in update mode.

**Comment by keirf** [comment]
You should copy the .upd file from the Greaseweazle distribution to root of your usb stick.

**Comment by shindoja** [comment]
Says e56 for 1 sec then e01

**Comment by keirf** [comment]
What file(s) did you copy to your USB drive?

**Comment by shindoja** [comment]
[greaseweazle-firmware-1.3.upd.zip](https://github.com/keirf/flashfloppy/files/11012427/greaseweazle-firmware-1.3.upd.zip)

**Comment by shindoja** [comment]
And i also tried to flash the gotek with the .hex file but the gotek did not lite up after. Illl try the f1 and f7 version

**Comment by shindoja** [comment]
![Screen Shot 2023-03-19 at 6 16 30 PM](https://user-images.githubusercontent.com/118574645/226212969-38184d01-2e2a-4dfb-bb52-31303360d203.png)

**Comment by keirf** [comment]
Why are you using Greaseweazle firmware files? You want FlashFloppy right?

**Comment by shindoja** [comment]
Those are the files on the root of usb stick 000 is the roland 2.24

**Comment by shindoja** [comment]
Is the stick should be in masterboot or partition guid ?

**Comment by keirf** [comment]
Recent firmware supports both mbr and gpt.

**Comment by shindoja** [comment]
ok

**Comment by keirf** [comment]
I've heard no success loading s760 samples. What would be useful is a s760 sample disk layout. Eg an hfe image file formatted on s760. Or areal sample disk dumped using something like Greaseweazle. Then we could see the actual disk layout.

**Comment by shindoja** [comment]
None of the greaseweaszle work on my Gotek :(

**Comment by shindoja** [comment]
I mean the firmware

**Comment by keirf** [comment]
Of course not. Greaseweazle doesn't run on Gotek.

**Comment by shindoja** [comment]
I dont have any original disk

**Comment by keirf** [comment]
Let me ask around.

**Comment by shindoja** [comment]
Ok thanks a lot !

**Comment by shindoja** [comment]
Its funny cause the s760 always re-create a new IMAGE_A.CFG file on the usb stick each time a boot it with it.

**Comment by keirf** [comment]
Yes it uses that to remember last image loaded.

**Comment by shindoja** [comment]
There's developpement ! I put the pin 2 and 34 in the ff.CFG file and it loaded 1 .IMG file but says : please insert disk 2 and then er and i have to cancel.

Also the other disk image say : ' unformated ' and ' not a sound disk '. I tried to format but change nothing.

This is my ff.CFG : 

 host = unspecified
 interface = jppc-hdout
 pin02 = rdy
 pin34 = chg

I'll try ibmpc-hdout  :) At least i know everything it good physically with the Gotek!

**Comment by shindoja** [comment]
[002.IMG.zip](https://github.com/keirf/flashfloppy/files/11018165/002.IMG.zip)
[DISK3.IMG.zip](https://github.com/keirf/flashfloppy/files/11018166/DISK3.IMG.zip)

The DISK3.IMG load but say error at the end ( feels like its too big or something )
The 002.IMG it say ' not sound disk '

**Comment by shindoja** [comment]
I'll try dens on pin 2

**Comment by daftpunker** [comment]
Hi all, I'm trying to save a sample from s760 on the gotek image senza successo. The gotek is correctly installed (jumper setting on JC and S0) in fact the OS on the s760 loads correctly, the problem occurs during the saving phase, first it asks to format the disk, then it starts writing, but in the end either a "floppy write error" error arrives or in other cases it hangs loading and crashing, so I'm forced to turn the s760 off and on again. I have gotek on roland S760 and on Roland dj70 are very similar, in fact I have the same problem with both. The s760 gotek has firmware 3.35 The dj70 gotek has firmware e 3.36. I enclose images of the Roland S760, the tests were done both on image 1.44 and 720 and screen of the usb stick and folders and images. I also tried 3 different stick usb with the same result. Can someone help me? 
![01](https://user-images.githubusercontent.com/128594378/226885636-887509e0-e3f2-4c1d-a4b2-6c4493acff1e.png)
![02](https://user-images.githubusercontent.com/128594378/226885795-6b606c1b-085f-42c0-95be-3dd6f8c3325a.png)
![03](https://user-images.githubusercontent.com/128594378/226885951-7e237e93-ff09-4e17-aa88-20ab0edb8d6d.png)
![04](https://user-images.githubusercontent.com/128594378/226886110-5ec932d3-37b6-4424-a734-b42017e0f6bc.png)
![05](https://user-images.githubusercontent.com/128594378/226886240-49fca607-d909-494a-8dc3-4689005f18e4.png)
![crash 01](https://user-images.githubusercontent.com/128594378/226886717-fda8a405-7cfc-4a40-876b-2264ad4d7e7b.png)
![crash 02](https://user-images.githubusercontent.com/128594378/226886765-e350e765-19d1-4f0f-b3ff-8d88683d6b81.png)

**Comment by daftpunker** [comment]
Hi all, I'm trying to save a sample from s760 on the gotek image senza successo. The gotek is correctly installed (jumper setting on JC and S0) in fact the OS on the s760 loads correctly, the problem occurs during the saving phase, first it asks to format the disk, then it starts writing, but in the end either a "floppy write error" error arrives or in other cases it hangs loading and crashing, so I'm forced to turn the s760 off and on again. I have gotek on roland S760 and on Roland dj70 are very similar, in fact I have the same problem with both. The s760 gotek has firmware 3.35 The dj70 gotek has firmware e 3.36. I enclose images of the Roland S760, the tests were done both on image 1.44 and 720 and screen of the usb stick and folders and images. I also tried 3 different stick usb with the same result. Can someone help me? 
![01](https://user-images.githubusercontent.com/128594378/226885636-887509e0-e3f2-4c1d-a4b2-6c4493acff1e.png)
![02](https://user-images.githubusercontent.com/128594378/226885795-6b606c1b-085f-42c0-95be-3dd6f8c3325a.png)
![03](https://user-images.githubusercontent.com/128594378/226885951-7e237e93-ff09-4e17-aa88-20ab0edb8d6d.png)
![04](https://user-images.githubusercontent.com/128594378/226886110-5ec932d3-37b6-4424-a734-b42017e0f6bc.png)
![05](https://user-images.githubusercontent.com/128594378/226886240-49fca607-d909-494a-8dc3-4689005f18e4.png)
![crash 01](https://user-images.githubusercontent.com/128594378/226886717-fda8a405-7cfc-4a40-876b-2264ad4d7e7b.png)
![crash 02](https://user-images.githubusercontent.com/128594378/226886765-e350e765-19d1-4f0f-b3ff-8d88683d6b81.png)

**Comment by daftpunker** [comment]
https://youtu.be/-MvFgu-2ktM

https://youtu.be/nnP3Vx2FgEk

**Comment by cidermassive** [comment]
Can you load these images into your DJ70? 
[DJ70-DemoDisks.zip](https://github.com/keirf/flashfloppy/files/11045021/DJ70-DemoDisks.zip)

If not, let me know and I will see what settings I have in my FF.CFG

**Comment by daftpunker** [comment]
Hi cidermassive, thanks but it didn't work, it won't load the demos you sent me.

**Comment by daftpunker** [comment]
keirf, thanks to you too for the answer, on the s760 it only loads the operating system without any problem. Then I try to export by saving a sample on gotek but after waiting for the usual formatting of the blank and started writing it returns at the end of the error

**Comment by daftpunker** [comment]
I have made various tests, I am attaching 2 zip folders that precisely reproduce the contents of the 2 USB sticks  s760 and dj70
[S760.zip](https://github.com/keirf/flashfloppy/files/11066012/S760.zip)
[DJ70.zip](https://github.com/keirf/flashfloppy/files/11066013/DJ70.zip)

**Comment by keirf** [comment]
Low RAM 415 based Goteks may have insufficient buffering to format an image.

**Comment by daftpunker** [comment]
Could you please check the two folders I attached earlier? Are the folder organization and cfg, hfe and img files in order as they should? Can you test them on yours to see if they work for you?

**Comment by cidermassive** [comment]
thanks for the files.....they are all the same demo disk file....with different names...was able to load all there here, blank,dska0002 and dj70 demo disk

i have formatted your blank.img on my dj70 

i am using ff 3.16 and also i believe one of the original models with larger processor....

attaching the dj70 formatted disk and my ff.cfg for your reference

[DJ70.zip](https://github.com/keirf/flashfloppy/files/11066522/DJ70.zip)

**Comment by cidermassive** [comment]
if it helps @keirf i can set up logging and attach the file

**Comment by daftpunker** [comment]
> @daftpunkergrazie per i file.....sono tutti lo stesso file del disco demo....con nomi diversi...è stato in grado di caricarli tutti e tre qui, vuoto,dska0002 e disco demo dj70
> 
> ho formattato il tuo blank.img sul mio dj70
> 
> sto usando ff 3.16 e credo anche uno dei modelli gotek originali con il processore più grande ....
> 
> allegando il disco formattato dj70 e il mio ff.cfg come riferimento
> 
> [DJ70.zip](https://github.com/keirf/flashfloppy/files/11066522/DJ70.zip)

@cidermassive ----->   **Your zip worked!!! GREAT!!! THANK YOU 10000000000000!!!**

**Comment by cidermassive** [comment]
Fantastic!

Can you also check if this file works with the S760?

**Comment by daftpunker** [comment]
@cidermassive 
Perfect, :) I also tried it on the S760 and.... IT WORKS!!! 

The "BlankDiskFormattedbyDj70" image works!!! ;)

**Comment by daftpunker** [comment]
I am grateful to you!

**Comment by cidermassive** [comment]
Enjoy the lovely sound of those Roland Samplers!

We are all very greatful to @keirf !!!

**Comment by keirf** [comment]
In the end, did this work with jumper at position S0 only? As well as the correct interface/pin setting in FF.CFG?

**Comment by daftpunker** [comment]
Hi all, I am writing to you again because other problems have appeared.
Actually I have n.3 Roland S760 and n.1 Roland DJ70
On each Roland I mounted its own Gotek respectively:
	Gotek	firmware	Jumper
Roland S760 (1)	SFRKC30AT 3	3.35 – 32KB	S0
Roland S760 (2)	SFRKC30AT 2	3.36 – 32KB	S0
Roland S760 (3)OP	SFRKC30AT 2	3.36 – 32KB	S0
Roland DJ70	SFRKC30AT 2	3.36 – 32KB	S0 – JC
![image](https://user-images.githubusercontent.com/128594378/228773877-1d48be7d-a749-4d3a-bedd-a10eac1d4a68.png)


Previously successful tests had been on these 2 machines:
	Gotek	firmware	Jumper
Roland S760 (1)	SFRKC30AT 3	3.35 – 32KB	S0
Roland DJ70	SFRKC30AT 2	3.36 – 32KB	S0 – JC
![image](https://user-images.githubusercontent.com/128594378/228773948-507889e0-6ae1-4782-82d3-309255cb6665.png)


Next I tried the same usb stick on the 2 remaining S760s and it didn't work:
	Gotek	firmware	Jumper	Load OS	Samples Load/Save
Roland S760 (2)	SFRKC30AT 2	3.36 – 32KB	S0	OK	KO
Roland S760 (3)OP	SFRKC30AT 2	3.36 – 32KB	S0	OK	KO
![image](https://user-images.githubusercontent.com/128594378/228774044-e7782b4c-83ad-4b87-9e79-1cc65f39484e.png)

So I tried to move the SFRKC30AT3 floppy swapping it on one of the 2 rolands and the test confirmed that the SFRKC30AT3 continued to work even on another S760 while the SFRKC30AT2 continued not to work even on the s760 where the SFRKC30AT 3 worked.
	Gotek	firmware	Jumper	Load OS	Samples Load/Save
Roland S760 (1)	SFRKC30AT 2	3.36 – 32KB	S0	OK	KO
Roland S760 (2)	SFRKC30AT 2	3.36 – 32KB	S0	OK	KO
Roland S760 (3)OP	SFRKC30AT 3	3.35 – 32KB	S0	OK	OK
Roland DJ70	SFRKC30AT 2	3.36 – 32KB	S0 – JC	OK	OK
![image](https://user-images.githubusercontent.com/128594378/228774143-da64c6eb-6110-4352-ab73-7b4af5efad8c.png)


Ah checking well on S760 the jumper positioned on JC causes the failure to load the OS. The only way to load OS is to place a single Jumper on S0.
	Gotek	firmware	Jumper	Load OS	Samples Load/Save
Roland S760 (1)	SFRKC30AT 3	3.35 – 32KB	S0	OK	OK
Roland S760 (2)	SFRKC30AT 2	3.36 – 32KB	S0 – JC	KO	-
Roland S760 (3)OP	SFRKC30AT 2	3.36 – 32KB	S0 – JC	KO	-
Roland DJ70	SFRKC30AT 2	3.36 – 32KB	S0 – JC	OK	OK
![image](https://user-images.githubusercontent.com/128594378/228774258-23fcd718-85b8-4e80-9291-b22e8a14b7a7.png)

What do you think is not right?
I am again attaching the content of the flash drive and below I summarize a diagram of the various trials:

![image](https://user-images.githubusercontent.com/128594378/228774530-e1f23bac-4af2-4324-a479-89c61641e4e6.png)


Many thanks for your attention

**Comment by daftpunker** [comment]
[S760-DJ70 OK.zip](https://github.com/keirf/flashfloppy/files/11109447/S760-DJ70.OK.zip)

Stick USB

**Comment by keirf** [comment]
Does the USB stick still have the required FF.CFG on it? Because you could remove that file, and the original working AT3 Gotek would continue to work, because the FF.CFG configuration gets cached within the Gotek. But then the AT2 Goteks would not work, because they have not seen the required FF.CFG file.

You might also upgrade/downgrade the AT2 Goteks to the same firmware version as the AT3.

Basically you want everything as identical as possible to the working Gotek.

**Comment by daftpunker** [comment]
Hi Keirf

AT3 remained in the stick the FF.CFG file works so I didn't make any changes to the stick.
AT2 I tried the same stick that I have attached above with the results described above.
I attach the error on trying to save a sample with this configuration:
![image](https://user-images.githubusercontent.com/128594378/228781547-e19928c2-e43e-43f1-8531-24c497b73289.png)
![image](https://user-images.githubusercontent.com/128594378/228781592-7aea4590-7ff1-4d35-8ae0-5f53c5e10e3a.png)

To be able to update AT2 Gotek to the latest firmware (v3.39) gladly, but I have to see some tutorials I'm not so expert

**Comment by keirf** [comment]
Download the v3.39 zip file. Copy the FF_Gotek*.upd file contained within to the root of your USB drive. 

With USB drive ejected, press the rotary encoder on your AT" to bring up main menu. Rotate to select "Firmware Update". Press the encoder a few times until the displays says "[USB]". Now insert the USB drive. You should automatically update to v3.39.

**Comment by daftpunker** [comment]
@keirf 

I performed what you told me on S760 (AT2):

1 I extracted the zip folder and copied the "FF_Gotek-3.39.upd" file to the root of the USB stick.
![image](https://user-images.githubusercontent.com/128594378/228804756-7d0a6ad1-8f85-44c9-a914-4c368aa91f6d.png)

2 I rotated the knob selecting firmware upgrade and pressing several times until "USB" came out

3 I inserted the USB stick containing the "FF_Gotek-3.39.upd" file in the root, the display showed me the update to 3.39 but in the end error 34 appeared
![image](https://user-images.githubusercontent.com/128594378/228804569-e7f3be66-00d4-457c-b252-c8eacd44ce09.png)

4 I extracted the stick and the display indicated 3.39 as firmware

5 I inserted the USB stick with the contents that work in the AT3 but the saving test always returned the same error
![image](https://user-images.githubusercontent.com/128594378/228804393-e409eca7-7801-4b6f-af7d-809d7ca58241.png)

**Comment by keirf** [comment]
Seems you updated okay. But I think there is a configuration error somewhere. Like I said earlier: Given the correct FF.CFG you should be using, the JC jumper should have no effect on Gotek behaviour. Since you report that it does have an effect, your configuration seems to be incorrect.

**Comment by daftpunker** [comment]
These are the 2 Goteks compared:
![IMG_20230326_230157](https://user-images.githubusercontent.com/128594378/228808824-4eb04b12-8b5e-4f91-994d-41bdef29b426.jpg)
![Uploading IMG_20230326_230155.jpg…]()

**Comment by daftpunker** [comment]
AT2: 
![IMG_20230326_230234](https://user-images.githubusercontent.com/128594378/228809023-e9b62be0-c800-4a24-bc93-ad926995a8c4.jpg)

![IMG_20230326_230348](https://user-images.githubusercontent.com/128594378/228809179-c4c78fd7-c236-4840-8fac-e353e6a6b2b3.jpg)


![IMG_20230326_230354](https://user-images.githubusercontent.com/128594378/228809281-d9d9b45b-0674-4974-9c11-750db67e62d4.jpg)

**Comment by keirf** [comment]
The hardware setup of both Goteks looks fine. This would be a software configuration difference.

On both AT2 and AT3:
1. Remove USB drive
2. Press rotary encoder to enter Main Menu
3. Rotate to select "Factory Reset". Press the rotary encoder twice.
4. Display should say "Reset Flash Configuration" and then back to main screen.

Now, as usual, with the exact same USB drive test the AT2 and AT3 in your S760. Do they behave at all differently now?

**Comment by daftpunker** [comment]
Factory reset done. When trying to save a sample the same error as above appears

**Comment by daftpunker** [comment]
And if we tried to install on gotek AT2 Firmware 3.35 equal to the one installed on ful AT3?

**Comment by daftpunker** [comment]
I think I'll have to do the same procedure you told me for 3.39

Firmware version 3.35 correct is this?

https://github.com/keirf/flashfloppy/releases/tag/v3.35


![image](https://user-images.githubusercontent.com/128594378/228814241-fe229057-8fa7-47ab-8e65-1efa44b3f568.png)

**Comment by keirf** [comment]
So you factory reset both and the AT3 worked but the AT2 didn't?

You can indeed downgrade the AT2 to 3.35, no problem.

**Comment by keirf** [comment]
Remove the old FF_Gotek*.upd file from the USB drive before copying across the 3.35 one.

**Comment by daftpunker** [comment]
Yes, I deleted the 3.39 update first and copied the 3.35 update to the USB root. Unfortunately nothing has changed, even with firmware 3.35 the same error appears when trying to save a sample. I also tried doing a factory reset but nothing changed

**Comment by daftpunker** [comment]
On AT3 I didn't touch anything as it works as configured. The tests I'm running on the AT2 are the ones that are making me damn

**Comment by keirf** [comment]
What happens if you put jumper JC on the AT2 now?

**Comment by daftpunker** [comment]
Tried with firmware 3.39 and Jumper in S0 and JC does not load OS. It basically initializes the process to load OS and crashes just before it starts loading countdown, reboots itself, S760 boot screen and so on

**Comment by keirf** [comment]
So the situation is that with the FF.CFG you report that you are using, it is impossible for jumper JC to have an effect. Jumper JC only affects pins 2 & 34 configuration, when `interface = jc` in FF.CFG, and `pin02 = auto` and `pin34 = auto`. Since you have overrides for both `pin02` and `pin34` in your FF.CFG, jumper JC can have no effect.

So, since jumper JC *does* have an effect (it stops the OS from loading), it tells me that your FF.CFG is not being read, or differs somehow from the one you report that you are using.

Compare with the working AT3:
1. If you installed jumper JC there, I would bet that it would have no effect (AT3 would still work ok)
2. If you factory reset the AT3, I would bet that it will stop working and behave just like the AT2.

**Comment by shindoja** [comment]
HI again ! Hope you doing well :)
My s760 manage to format one disk but i cant understand what is it.
I just want to be able to make a new disk to put samples on it and i cant since HcX does not let me to.
Each combination i try on the HcX say ' not formatted ' on the s760.
Can you guys tell me what is it then a can make new empty ones ? It says ' Unknown ' in the track analyser

[Drumz.hfe.zip](https://github.com/keirf/flashfloppy/files/11216702/Drumz.hfe.zip)

Thanks a lot  !

**Comment by daftpunker** [comment]
Hello, I understand that you need to create blanks for s760? I honestly copy the virgin blank file from my pc and everything works. Try loading this blank let me know
[Blank.zip](https://github.com/keirf/flashfloppy/files/11283364/Blank.zip)

**Comment by shindoja** [comment]
Oh ! It worked right on spot ! So you think i could put my personal samples file in there ? I never did that on PC ! If you could tell me the procedure format etc. Thanks so much !

**Comment by keirf** [comment]
@daftpunker Can I add your blank formatted image to the image library? https://github.com/keirf/flashfloppy-images

Formatted images for other systems are also welcome!!

**Comment by shindoja** [comment]
Ok I read too fast this morning haha So you just copy the blank file but you cannot modify it from the PC and put some samples in it I believe. I think a have to buy a software for that :(

**Comment by daftpunker** [comment]
@shindoja I attached everything you need to be able to convert your wav files in format compatible with your s760. Consider that the file that comes out to be read by your s760 will need to go to the DISK section then go to the submenu where we find LOAD, SAVE etc. and select the "convert" command, you will see that the sample will be converted to the compatible format and everything will work! Good fun

**Comment by daftpunker** [comment]
[Uploading WAV to Roland W30 v0.3.zip…]()
:)

**Comment by shindoja** [comment]
Oh ! Thanks so much ! That would absolutely save my day ! But when I click on the link it brings me back to the top of this page tho :)

**Comment by shindoja** [comment]
hi @daftpunker ! Im quite ready to start recording now haha ! Im really into get that attachement file but the link only brings me to the top of the page hehe ( maybe something i dont understand also ) But if you could send me the files that would be marvellous ! Thank you :)

**Comment by keirf** [comment]
The link is just a link to this ticket. You will need @shindoja to post the correct link.

**Comment by shindoja** [comment]
> @shindoja I attached everything you need to be able to convert your wav files in format compatible with your s760. Consider that the file that comes out to be read by your s760 will need to go to the DISK section then go to the submenu where we find LOAD, SAVE etc. and select the "convert" command, you will see that the sample will be converted to the compatible format and everything will work! Good fun

Yes I was vigorously looking for the attached file mention here :) By @Daftpunker !

**Comment by daftpunker** [comment]
Hi guys, sorry I've been out I read now. I try again attach the program to convert wav files into images compatible with s760
[OK Wav to roland.zip](https://github.com/keirf/flashfloppy/files/11542061/OK.Wav.to.roland.zip)

**Comment by keirf** [comment]
^ Yay! Thanks!
